scripts/qa_crowbarsetup.sh

```diff
@@ -3412,6 +3412,17 @@ function get_cluster_name
     echo $cluster
 }
 
+function get_controller
+{
+    local element
+    if [[ $want_nova_proposal = 1 ]]; then
+        get_novacontroller
+    elif [[ $want_neutron_proposal = 1 ]]; then
+        get_neutron_server_node
+        novacontroller=NEUTRON_SERVER
+    fi
```

Comment:
Since you don't have an `else`, and the previous behavior was the first branch, why not rearrange this to
```
if [[ $want_nova_proposal != 1 ]]; then
  get_neutron_server_node
  novacontroller=NEUTRON_SERVER
fi
```

Reply:
i hope you meant this.

Reply:
Yes :+1: :smile: